src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [hasLoadedOnce, setHasLoadedOnce] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        const updatedPeople = data.map(person => ({
+          ...person,
+          mother: data.find(p => p.name === person.motherName),
+          father: data.find(p => p.name === person.fatherName),
+        }));
+
+        setPeople(updatedPeople);
+      })
+      .catch(() => setIsError(true))
+      .finally(() => {
+        setIsLoading(false);
+        setHasLoadedOnce(true);
+      });
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          <div className="column is-7-tablet is-narrow-desktop">
+            {!!people.length && <PeopleFilters />}
+          </div>
+
+          <div className="column">
+            <div className="box table-container">
+              {isLoading && <Loader />}
+              {isError && (
+                <p data-cy="peopleLoadingError" className="has-text-danger">
+                  Something went wrong
+                </p>
+              )}
+
+              {hasLoadedOnce && !isLoading && !isError && !people.length && (
+                <p data-cy="noPeopleMessage">
+                  There are no people on the server
+                </p>
+              )}
+
+              {!isError && people.length > 0 && <PeopleTable people={people} />}
```

Comment:
This is a code style issue. It's better to use explicit checks instead of coercing values to boolean. In this case, it's better to check if the 'people' array is not empty explicitly: 'people.length > 0'.